addons/web_editor/static/src/js/common/ace.js

```diff
@@ -1112,6 +1117,11 @@ var ViewEditor = Widget.extend({
     _onEditAnywayClick: function () {
         this.$warningMessage.addClass('d-none');
     },
+
+    _onDontAskAgainClick: function () {
```

Comment:
This last commit should be squashed with the main work, probably does not deserve a commit of its own :)